### DISCUSSION TITLE
[548430] 深岩银河 Deep Rock Galactic（本体）

### DISCUSSION THREAD

**Discussion by Dew778** [pjy612/SteamManifestCache#384] [Share]
`
appid：548430
`

```
548431;73ef10dc344cb7fb34be23aea35b20fbb2b32b0db50650919c7e3d731e759aad
```

[548430_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13972714/548430_config.vdf.txt)
[[548430] 深岩银河 Deep Rock Galactic（本体）清单.zip](https://github.com/pjy612/SteamManifestCache/files/13973041/548430.Deep.Rock.Galactic.zip)

Code：7337BCEA697F2EBE
Email:3394748293@qq.com

**Comment by ZEERDEER** [comment]
标题格式 #26

**Reply by Dew778** [reply]
已修改
